.github/workflows/ci.yml

```diff
@@ -51,6 +51,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-go-for-project
+      - uses: cachix/install-nix-action@v30
+        with:
```

Comment:
why is this github token is needed while it was not needed in the previous version of this file?